fem/fespace.hpp

```diff
@@ -921,6 +921,10 @@ class FiniteElementSpace
    { return mesh->GetBdrElementType(i); }
 
    /// Returns ElementTransformation for the @a i-th element.
+   /// @note The returned object is owned by the associated @a Mesh
+   /// and is shared, i.e., calling this function resets pointers
+   /// obtained from previous calls.
+   /// Also, this pointer should @b not be deleted by the caller.
```

Comment:
```suggestion
   /// @note The returned pointer references an object owned by the associated 
   /// @a Mesh that will be modified by other calls to `GetElementTransformation`.
   /// As such, this pointer should @b not be deleted by the caller.
```
While I imagine the language here is chosen to be consistent with the comments for `Mesh::GetElementTransformation`, i.e., https://github.com/mfem/mfem/blob/master/mesh/mesh.hpp#L1736, I find them both misleading.  The danger here is that the object the return pointer references will be modified by `Mesh` on each call to `Mesh::GetElementTransformation`.